src/XMakeTasks/Microsoft.CSharp.targets

```diff
@@ -17,6 +17,15 @@ Copyright (C) Microsoft Corporation. All rights reserved.
 
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
    <!-- 
+        We are doing a cross-targeting traversal build if there is no list of target frameworks specified
+        nor is there a current target framework being built individually. In that case, this import is
+        redirected to Microsoft.Common.Traversal.targets.
+   -->
```

Comment:
Am I right that this needs to be defined in whatever is the top-level import, because the goal is to short-circuit most of the build logic for the outer build?